sample-apps/node/frontend-service/index.js

```diff
@@ -77,19 +77,30 @@ app.get('/remote-service', (req, res) => {
   request.end();
 });
 
+// The following logic serves as the async call made by the /client-call API
+let makeAsyncCall = false;
+setInterval(() => {
+  if (makeAsyncCall) {
+    makeAsyncCall = false;
+    console.log('Async call triggered by /client-call API');
+
+    const request = http.get('http://local-root-client-call', (rs) => {
+      rs.setEncoding('utf8');
+      rs.on('data', (result) => {
+        res.send(`GET local-root-client-call response: ${result}`);
+      });
+    });
+    request.on('error', (err) => {}); // Expected
+    request.end();
+  }
+}, 5000); // Check every 5 seconds
+
 app.get('/client-call', (req, res) => {
-  // Immediately respond to the client without waiting for the HTTP request to complete
-  res.status(202).send('/client-call called successfully');
-  console.log('/client-call called successfully')
+  res.send('/client-call called successfully');
+  console.log('/client-call called successfully');
 
-  const request = http.get('http://local-root-client-call', (rs) => {
-    rs.setEncoding('utf8');
-    rs.on('data', (result) => {
-      res.send(`/client-call response: ${result}`);
-    });
-  });
-  request.on('error', (err) => {});
-  request.end();
+  // Trigger async call to generate telemetry for InternalOperation use case
+  makeAsyncCall = true;
```

Comment:
If you just want to trigger the InternalOperation, you don't need this. The variable couples the the sync and async calls together.

Reply:
It's also to ensure the trace segments for the initial call and the async call are connected.